pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -126,6 +131,26 @@ func moveVFtoContainerNS(vfNetDevice string, containerID string, containerNetNS
 			return fmt.Errorf("failed to configure IP address for container %s: %v", containerID, err)
 		}
 		klog.V(2).Infof("ipam.ConfigureIface result: %+v, err: %v", result, err)
+		// For VF interface in a Pod or container namespace (without VF Representator), configure policy rules to isolate the route lookup specific to the secondary network.
```

Comment:
Sorry, I actually meant "Pod namespace" or "container namespace" here.

Reply:
thanks. this section was removed with the updated implementation.